source/load-testing-dojot-platform.rst

```diff
@@ -295,9 +293,7 @@ Before running the script, we need to initialize the Locust master. Inside Locus
 
   docker-compose -f Docker/docker-compose-master.yml up
 
-
-After its initialization, initialize the ``generate_certs`` script docker compose in the **primary**
-machine and go inside it:
+After its initialization, initialize the docker compose ``generate_certs`` script in the **primary** machine and enter in it:
```

Comment:
> [...] initialize the docker compose \``generate_certs\`` script in [...]

This is exactly how I mentioned in the previous comment :+1:

Reply:
Great, thanks